spec/acceptance/suites/default/redis_loadmodule.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper_acceptance'
```

Comment:
Add unit tests instead of acceptance.
Acceptance are a little bit overkill for checking the config IMO